Common/src/main/java/com/tabnineCommon/chat/commandHandlers/InitHandler.kt

```diff
@@ -15,18 +18,24 @@ data class InitPayload(
     private val colors: MutableMap<String, String>,
     private val fontSize: Int,
     private val isTelemetryEnabled: Boolean,
-    private val serverUrl: String?
+    private val serverUrl: String?,
+    private val workspaceFolders: List<String>
 )
 
 class InitHandler(gson: Gson) : ChatMessageHandler<Unit, InitPayload>(gson) {
     override fun handle(payload: Unit?, project: Project): InitPayload {
+        val workspaceFolders = AsyncAction(ActionPermissions.READ).execute {
+            ProjectRootManager.getInstance(project).contentRootUrls
+        }.get()
+
         return InitPayload(
             "ij",
             EditorColorsManager.getInstance().isDarkEditor,
             readColorPalette(),
             EditorColorsManager.getInstance().globalScheme.getFont(EditorFontType.PLAIN).size,
             isTelemetryEnabled(),
-            getServerUrl()
+            getServerUrl(),
+            workspaceFolders
```

Comment:
why are we send the workspace folders with the init?

Reply:
I mean, this list could change after the chat is open.

Wouldn't it be better to sk for it when you issue a retrieval query?

Reply:
huh, it was just some draft i had when i played around with the workspace folders last week, i didn't notice i hadn't rolled it back.